app/routes/index.js

```diff
@@ -11,22 +11,39 @@ export default Route.extend(PlayParamMixin, {
   googleAds:  service(),
   classNames: ['home'],
   dj: service(),
+  fastboot: service(),
+  metadata: service(),
 
   model() {
     get(this, 'googleAds').doTargeting();
 
-    return RSVP.hash({
-      wqxrHome: this.store.findRecord('bucket', 'wqxr-home').then(b => {
-        return {
-          featuredItems: b.get('bucketItems').slice(0, 9),
-          otherItems: b.get('bucketItems').slice(9)
-        };
-      }),
-      wartopChunk: this.store.findRecord('chunk', 'wqxr-wartop-home').catch(()=>''),
-      membershipChunk: this.store.findRecord('chunk', 'wqxr-membership-home').catch(() => ''),
+    if (this.get('fastboot.isFastBoot')) {
```

Comment:
It'd be nice to DRY up this section a bit, maybe with an approach like this:

```
  let hash = {
    wqxrHome: this.store.findRecord('bucket', 'wqxr-home').then(b => {
      return {
        featuredItems: b.get('bucketItems').slice(0, 9),
        otherItems: b.get('bucketItems').slice(9)
      };
    }),  
  };

  if (!this.get('fastboot.isFastBoot')) {
    hash.wartopChunk = this.store.findRecord('chunk', 'wqxr-wartop-home').catch(()=>''),
    hash.membershipChunk = this.store.findRecord('chunk', 'wqxr-membership-home').catch(() => ''),
  }

  return RSVP.hash(hash);
```

Reply:
good point, i'll do this!